recipes-dom0/dom0-image-weston/files/meta-xt-extra/recipes-kernel/linux/linux-renesas_%.bbappend

```diff
@@ -0,0 +1,46 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+RENESAS_BSP_URL = "git://github.com/xen-troops/linux.git"
```

Comment:
no, this is the way it is defined in meta-rcar-gen3/recipes-kernel/linux/linux-renesas_4.9.bb